### PR TITLE
fix: use Automation Orchestrator in API specs, CLI, and container labels

### PR DESCRIPTION
**Pull request by Qalthos**
## Description

Replace standalone "Orchestrator" with "Automation Orchestrator" as the downstream product name across all user-facing and API-visible surfaces. The partial rename in PR #261 substituted "Syntara" with "Orchestrator" but we require the full downstream name "Automation Orchestrator".

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)

## Changes Made

- [x] OpenAPI sub-spec titles and descriptions (19 domain specs): `Orchestrator` → `Automation Orchestrator`
- [x] Workflow sub-specs: contact name, WorkflowDefinition description, credential UUID field descriptions
- [x] FastAPI app title (`main.py`) and spec generation tools (`export_openapi.py`, `bundle_openapi.py`)
- [x] Python model docstrings/field descriptions: `workflow_definition.py`, `aap/models/queries.py`, `identity_providers/models/`, `metrics/`
- [x] API client templates (jinja), `pyproject.toml`, `README.md`
- [x] Dynamic API CLI tool name: `orchestrator_cli/__init__.py`, `auth.py`, `cli/pyproject.toml`
- [x] Admin CLI renamed: `orchestrator-admin` → `automation-orchestrator-admin` with `ao-admin` alias (no backward-compat alias)
- [x] Config/OTEL: added `test_otel_service_name_derives_from_product_name` verifying `APP_PRODUCT_NAME` drives the OTEL service name
- [x] Tekton CI labels: `DESCRIPTION_LABEL=Syntara UI` → `Automation Orchestrator UI` (4 files)
- [x] Compliance test: updated to assert `Automation Orchestrator` in spec title
- [x] API version bump: `1.3.0` → `1.3.1` (`constants.py` + regenerated bundles)
- [x] Docs: `backend/docs/authentication.md` — updated 13 admin CLI command references to the new command name
- [x] Regenerated: `openapi.yaml`, `openapi-public.yaml/.json`, `syntara_api_client/`, frontend contracts

## Out of Scope

These are intentionally unchanged — they use the upstream community name "Syntara" or are infrastructure identifiers not governed by downstream product naming:

- `COMMUNITY_APP_TITLE = 'Syntara'`, `<title>Syntara</title>` — upstream community name
- `Containerfile DESCRIPTION="Syntara UI"` — upstream
- Python/npm package names (`syntara`, `@syntara/*`) — upstream identifiers
- User-Agent headers (`Syntara-Workflow-Engine`) — upstream identity
- Environment variables (`SYNTARA_E2E_*`, etc.) — upstream
- Konflux namespace (`nexus-tenant`, `nexus-segment-write-keys`) — infra, requires admin coordination
- `orchestrator-cli` (dynamic API CLI) — not a supported product; its `~/.orchestrator/` token/spec cache is untouched. Note: the tool's own display name in help text/docstrings *was* updated (tool identity ≠ product-in-docs).
- `orchestrator_` Prometheus metric prefix — per Prometheus naming conventions (single-word app prefix); renaming would break existing dashboards/alerts
- `Agent Orchestrator` — component name (agentic subsystem), not a product name reference
- `OrchestratorPrometheusMetrics` — internal class name predating PR #261
- `X-Orchestrator-Client` HTTP header — internal wire protocol, breaking change if renamed
- `backend/docs/` — upstream documentation; "Syntara" as the upstream product name is left correct where appropriate. Edits were made only where warranted for accuracy (e.g., renamed admin CLI command references in `authentication.md`).

## Related Issues

Related to #261 (the partial rename this PR corrects)

## How to Test

1. Run `uv run pytest tests/unit/api/compliance/test_api_spec_naming.py -v` — 3/3 should pass
2. Start the backend and visit `/api_docs/v1/docs` — title should read "Automation Orchestrator API"
3. Run `orchestrator --help` — the dynamic API CLI help text should reference "Automation Orchestrator API"
4. Run `automation-orchestrator-admin --help` and `ao-admin --help` (with `APP_PRODUCT_NAME="Automation Orchestrator"`) — both admin CLI commands should reference "Automation Orchestrator"

## Backend Checklist

- [ ] I have run `make test` and all tests pass (in `backend/`)
- [ ] I have run `make lint` and code passes all checks
- [ ] I have run `make format` to ensure consistent formatting
- [ ] I have run `make typecheck` and all types are correct
- [x] I have added tests for new functionality
- [ ] Database migrations are included if schema changed
- [x] No sensitive information (keys, passwords, etc.) is included

## Frontend Checklist

- [ ] I have run `npm test` and all tests pass (in `frontend/`)
- [ ] I have run `npm run lint` and code passes all checks
- [x] I have run `npm run tsc` and there are no type errors
- [ ] I have added tests (unit / E2E) for new functionality
- [ ] Accessibility has been considered (semantic HTML, labels, keyboard navigation)
- [ ] Screenshots or screen recordings are attached for UI changes

## Visual Regression

- [ ] If this PR intentionally changes covered UI: I have commented `/update-screenshots` on this PR and confirmed the regenerated baselines look correct in the Files changed tab
- [ ] If this PR adds a new page/route: I have added an entry to `page-registry.ts` and run `/update-screenshots` to generate its baseline

## General Checklist

- [x] Code follows the project's coding conventions
- [x] I have updated relevant documentation
- [x] No secrets or credentials are included in this PR
- [ ] Breaking changes are documented with migration steps

### CODE REVIEW

**Review by skochay330 — APPROVED** [review]
Approving for frontend

File: backend/pyproject.toml [hunk 1]
```diff
@@ -139,7 +139,8 @@ orchestrator-cli = { path = "src/cli", editable = true }
 orchestrator-test-sdk = { path = "test-sdk", editable = true }
 
 [project.scripts]
-orchestrator-admin = "syntara.orchestrator_admin:app"
```
Comment (jewzaam) [inline backend/pyproject.toml:142]: #### I0: Admin CLI command `orchestrator-admin` removed with no alias, deprecation notice, or migration note

**Locations:**
- `backend/pyproject.toml:141-143`
- `backend/docs/authentication.md:316-318` (related)
- `backend/src/syntara/authz/seed.py:330` (related)
- `backend/tools/generate_secrets.sh:62` (related)

**Dimensions:** runtime_scope=service-internal, failure_mode=degraded-behavior, evidence_quality=demonstrated, trace_origin=entry-point, effort_to_fix=trivial

**Issue:** The production admin CLI's supported command name `orchestrator-admin` is removed outright and replaced by `automation-orchestrator-admin`/`ao-admin`, with no compatibility alias, no deprecation warning, and no migration note shipped anywhere in the PR (the PR's own 'General Checklist' item 'Breaking changes are documented with migration steps' is left unchecked).

**Why it matters:** This is a CLI surface used for privileged, incident-response-critical operations (account re-enable, password reset, session revocation) that the project's own docs describe as run 'inside the application pod' — i.e. by operators via kubectl exec or automation, exactly the kind of consumer this axis treats as broad. Narrowing the accepted command name without any transition path forces every existing caller to act at upgrade time with no warning.

**Suggested fix:** Add `orchestrator-admin = "syntara.orchestrator_admin:app"` back to `[project.scripts]` alongside the new names for at least one deprecation window, and have the entry point print a one-line deprecation notice to stderr when invoked under the old name (or, at minimum, record the rename in a CHANGELOG/migration note referenced from the PR).

File: backend/tests/unit/api/test_docs_endpoints.py [hunk 2]
```diff
@@ -358,6 +358,42 @@ def test_docs_redirect_to_api_docs(self, monkeypatch: pytest.MonkeyPatch) -> Non
...
+        assert translated["x-package"] == "syntara_api_client"
+        assert schema["info"]["title"] == "Syntara API"
+
+    def test_app_title_uses_configured_product_name(self, monkeypatch: pytest.MonkeyPatch) -> None:
+        """The runtime app and OpenAPI document use APP_PRODUCT_NAME."""
+        import importlib
+
+        import syntara.api.main as main_module
+
+        monkeypatch.setenv("APP_PRODUCT_NAME", "Automation Orchestrator")
```
Comment (jewzaam) [inline backend/tests/unit/api/test_docs_endpoints.py:385]: ##### S9: test_app_title_uses_configured_product_name's finally block forgets to reset APP_PRODUCT_NAME before reload

**Locations:**
- `backend/tests/unit/api/test_docs_endpoints.py:379-396`

**Dimensions:** runtime_scope=ci, failure_mode=confusion, evidence_quality=demonstrated, trace_origin=component, effort_to_fix=trivial

**Issue:** test_app_title_uses_configured_product_name sets APP_PRODUCT_NAME='Automation Orchestrator' and APP_ENABLE_API_DOCS='false', then reloads syntara.api.main. Its finally block restores neither: it re-sets APP_ENABLE_API_DOCS to 'false', the same value the test itself set, and never touches APP_PRODUCT_NAME. monkeypatch's own teardown runs only after the test function returns, so the finally block's cache_clear()+reload pair rebuilds the module with the override still in the environment. Its only effect is a redundant reload that re-applies the override.

**Why it matters:** After this test, main_module.app.title stays 'Automation Orchestrator API' and the get_settings cache holds the overridden Settings, for the rest of the process. Every sibling test in TestDocsEnabledWiring/TestProductionAppWiring resets the vars it touches before its final reload precisely to avoid that; this test reads as if it does the same while restoring nothing.

**Suggested fix:** Add `monkeypatch.delenv("APP_PRODUCT_NAME", raising=False)` to the `finally` block alongside the existing `APP_ENABLE_API_DOCS` reset, before `get_settings.cache_clear()` and `importlib.reload(main_module)`.

File: backend/tests/unit/core/config/test_config.py [hunk 3]
```diff
@@ -328,6 +328,13 @@ def test_product_name_rejects_invalid_characters(self, monkeypatch: pytest.Monke
         with pytest.raises(ValidationError, match="pattern"):
             Settings(_env_file=None)
 
+    def test_otel_service_name_derives_from_product_name(self, monkeypatch: pytest.MonkeyPatch) -> None:
```
Comment (jewzaam) [inline backend/tests/unit/core/config/test_config.py:331]: ##### S10: New otel_service_name derivation test only covers the unset-env-var path, missing the explicit-override-collides-with...

**Locations:**
- `backend/tests/unit/core/config/test_config.py:331-336`
- `backend/src/syntara/core/config/base.py:2010-2014` (related)

**Dimensions:** runtime_scope=ci, failure_mode=unclear, evidence_quality=inferred, trace_origin=local, effort_to_fix=trivial

**Issue:** The new test_otel_service_name_derives_from_product_name only exercises the case where APP_OTEL_SERVICE_NAME is unset; it does not test the case where a caller explicitly sets APP_OTEL_SERVICE_NAME to a value equal to the field's default ('syntara'), which the sentinel-based derivation logic in config/base.py cannot distinguish from 'not set' and would silently override.

**Why it matters:** This PR specifically added the derive-from-product-name test to verify the new naming behavior end to end; leaving out the sentinel-collision edge case means the test suite doesn't protect against the derivation logic clobbering an explicit operator override of the OTEL service name, which would be a silent misconfiguration in telemetry routing.

**Suggested fix:** Add a test that sets both APP_PRODUCT_NAME (non-default) and APP_OTEL_SERVICE_NAME='syntara' (matching the field default) and asserts the expected behavior, documenting/locking in whichever behavior is intended (explicit-wins vs. derived-wins).

File: backend/tests/unit/orchestrator_admin/test_orchestrator_admin_cli.py [hunk 4]
```diff
@@ -622,3 +630,23 @@ def test_help_lists_commands(self) -> None:
...
         assert "enable-user" in plain
         assert "reset-password" in plain
+
+    def test_help_uses_product_name(self, monkeypatch: pytest.MonkeyPatch) -> None:
+        """Admin CLI help text reflects APP_PRODUCT_NAME at import time."""
+        import importlib
+
+        import syntara.orchestrator_admin.__main__ as admin_main
+
+        monkeypatch.setenv("APP_PRODUCT_NAME", "Automation Orchestrator")
```
Comment (jewzaam) [inline backend/tests/unit/orchestrator_admin/test_orchestrator_admin_cli.py:640]: ##### S11: test_help_uses_product_name cleanup doesn't restore module to default state, despite comment claiming it does

**Locations:**
- `backend/tests/unit/orchestrator_admin/test_orchestrator_admin_cli.py:634-652`
- `backend/src/syntara/orchestrator_admin/__main__.py:37` (related)

**Dimensions:** runtime_scope=ci, failure_mode=confusion, evidence_quality=demonstrated, trace_origin=component, effort_to_fix=trivial

**Issue:** test_help_uses_product_name sets APP_PRODUCT_NAME and reloads the admin CLI module, then attempts to restore default state in `finally` by reloading again — but it neither clears the get_settings lru_cache nor resets the env var before that reload, so the module-level `_product_name` stays polluted with 'Automation Orchestrator' for the rest of the test session.

**Why it matters:** The comment claims the reload 'restores the module to its default state so other tests are unaffected' and it does not. The test also depends on an autouse fixture in conftest for its own setup to work, which is invisible at the call site — a reader copying this as a template for reload-based tests inherits both.

**Suggested fix:** In the `finally` block, call `monkeypatch.delenv("APP_PRODUCT_NAME", raising=False)` (or setenv to the default) and `get_settings.cache_clear()` before `importlib.reload(admin_main)`, matching the pattern already used in test_docs_endpoints.py.

**Review by jewzaam — CHANGES_REQUESTED** [review]

File: backend/src/syntara/schemas/aap/openapi.yaml [hunk 5]
```diff
@@ -12,14 +12,14 @@ info:
     without direct Ansible Automation Platform Controller access.
 
     Authentication to the Ansible Automation Platform Controller is resolved
-    server-side from the AAP Gateway integration (URL and TLS) and an Orchestrator
+    server-side from the AAP Gateway integration (URL and TLS) and an Syntara
```
Comment (sathiya06) [inline backend/src/syntara/schemas/aap/openapi.yaml]: **Nit: "an Syntara" — drop the article before the product name**

Reads wrong for default Syntara; "an Automation Orchestrator" only happens to work after replace. Prefer "…using Syntara credentials…" (no a/an before the replaceable token).

**Review by sathiya06 — APPROVED** [review]
## Code Review — Approved

HEAD `494f459`. Branding/CLI approach looks good overall. One optional nit below; not blocking.

### Nit
1. AAP proxy schema says "an Syntara" — drop the article before the product name so translation works for both Syntara and Automation Orchestrator.

### Notes
- jewzaam I0 appears addressed on HEAD (`orchestrator-admin` retained); S9–S11 look fixed. I0 thread still unresolved.
